packages/pigeon/example/app/ios/Runner/AppDelegate.swift

```diff
@@ -15,14 +15,14 @@ private class PigeonApiImplementation: ExampleHostApi {
   }
 
   func add(_ a: Int64, to b: Int64) throws -> Int64 {
-    if (a < 0 || b < 0) {
-      throw FlutterError(code: "code", message: "message", details: "details");
+    if a < 0 || b < 0 {
+      throw FlutterError(code: "code", message: "message", details: "details")
     }
```

Comment:
This is the changed line of code from the README

Reply:
Ah, thank you!